src/main/java/seedu/address/ui/ResultDisplay.java

```diff
@@ -7,7 +7,7 @@
 import javafx.scene.layout.Region;
 
 /**
- * A ui for the status bar that is displayed at the header of the application.
+ * A UI for the status bar that is displayed at the header of the application.
```

Comment:
Is it `An UI` or `a UI` ?

Reply:
It should be `A UI` because in both `user interface` and `UI` the word/abbreviation is pronounced as though it starts with the consonant `y`.